src/main/io/gps.c

```diff
@@ -521,50 +688,156 @@ static void ubloxSendPollMessage(uint8_t msg_id)
+        offset += ubloxAddValSet(&tx_buffer, CFG_NAVSPG_INFIL_NCNOTHRS, payload, offset); // 81
+
+        payload[0] = 0;
+        offset += ubloxAddValSet(&tx_buffer, CFG_NAVSPG_INFIL_CNOTHRS, payload, offset); // 86
+
+        ubloxSendConfigMessage(&tx_buffer, MSG_CFG_VALSET, offsetof(ubxCfgValSet_t, cfgData) + offset, true);
+    } else {
+        memset(&tx_buffer, 0, sizeof(ubxMessage_t));
+        tx_buffer.payload.cfg_nav5.mask = 0xFFFF;
+        tx_buffer.payload.cfg_nav5.dynModel = model == 0 ? 0 : model + 1; //no model with value 1
+        tx_buffer.payload.cfg_nav5.fixMode = 3;
+        tx_buffer.payload.cfg_nav5.fixedAlt = 0;
+        tx_buffer.payload.cfg_nav5.fixedAltVar = 10000;
+        tx_buffer.payload.cfg_nav5.minElev = 5;
+        tx_buffer.payload.cfg_nav5.drLimit = 0;
+        tx_buffer.payload.cfg_nav5.pDOP = 250;
+        tx_buffer.payload.cfg_nav5.tDOP = 250;
+        tx_buffer.payload.cfg_nav5.pAcc = 100;
+        tx_buffer.payload.cfg_nav5.tAcc = 300;
+        tx_buffer.payload.cfg_nav5.staticHoldThresh = 0;
+        tx_buffer.payload.cfg_nav5.dgnssTimeout = 60;
+        tx_buffer.payload.cfg_nav5.cnoThreshNumSVs = 0;
+        tx_buffer.payload.cfg_nav5.cnoThresh = 0;
+        tx_buffer.payload.cfg_nav5.staticHoldMaxDist = 200;
+        tx_buffer.payload.cfg_nav5.utcStandard = ubloxUTCStandardConfig_int[gpsConfig()->gps_ublox_utc_standard];
+
+        ubloxSendConfigMessage(&tx_buffer, MSG_CFG_NAV_SETTINGS, sizeof(ubxCfgNav5_t), false);
+    }
 }
 
-static void ubloxSendNAV5Message(uint8_t model)
+// *** temporarily disabled for testing ***
+// static void ubloxSendNavX5Message(void) {
+//     ubxMessage_t tx_buffer;
+// 
+//     if (gpsData.ubloxM9orAbove) {
+//         uint8_t payload[1];
+//         payload[0] = 1;
+//         size_t offset = ubloxValSet(&tx_buffer, CFG_ANA_USE_ANA, payload, UBX_VAL_LAYER_RAM); // 5
+// 
+//         ubloxSendConfigMessage(&tx_buffer, MSG_CFG_VALSET, offsetof(ubxCfgValSet_t, cfgData) + offset, true);
+//     } else {
+//         memset(&tx_buffer, 0, sizeof(ubxMessage_t));
+// 
+//         tx_buffer.payload.cfg_nav5x.version = 0x0002;
+// 
+//         tx_buffer.payload.cfg_nav5x.mask1 = 0x4000;
+//         tx_buffer.payload.cfg_nav5x.mask2 = 0x0;
+//         tx_buffer.payload.cfg_nav5x.minSVs = 0;
+//         tx_buffer.payload.cfg_nav5x.maxSVs = 0;
+//         tx_buffer.payload.cfg_nav5x.minCNO = 0;
+//         tx_buffer.payload.cfg_nav5x.reserved1 = 0;
+//         tx_buffer.payload.cfg_nav5x.iniFix3D = 0;
+//         tx_buffer.payload.cfg_nav5x.ackAiding = 0;
+//         tx_buffer.payload.cfg_nav5x.wknRollover = 0;
+//         tx_buffer.payload.cfg_nav5x.sigAttenCompMode = 0;
+//         tx_buffer.payload.cfg_nav5x.usePPP = 0;
+// 
+//         tx_buffer.payload.cfg_nav5x.aopCfg = 0x1; //bit 0 = useAOP
+// 
+//         tx_buffer.payload.cfg_nav5x.useAdr = 0;
+// 
+//         ubloxSendConfigMessage(&tx_buffer, MSG_CFG_NAVX_SETTINGS, sizeof(ubxCfgNav5x_t), false);
+//     }
+// }
```

Comment:
If this is not needed, remove it

Reply:
There will be a subsequent PR to discuss Assist Now Autonomous, at which point the code will either be restored, or deleted.